src/components/shared/modals/ProjectToolDrawerModal.tsx

```diff
@@ -159,7 +126,7 @@ export default function ProjectToolDrawerModal({
             <Trans>Transfer ownership</Trans>
```

Comment:
not related to your PR but could we wrap this in `span` to fix styling for loading state?

![Uploading Screen Shot 2022-04-13 at 6.36.12 am.png…]()